style.css

```diff
@@ -0,0 +1,3 @@
+h1{
+    color: red;
```

Comment:
Please ensure that you add a descriptive README file to your project - please use [this template](https://github.com/microverseinc/readme-template) but remember to customize it to your project.